serveradmin/dataset/create.py

```diff
@@ -70,6 +74,21 @@ def create_server(
         return server_id
 
 
+def access_control(user, server):
+    if not user or user.is_superuser:
```

Comment:
Applications are also using this.  I will develop access control for them, later.